card/mod/carrierwave/spec/lib/carrier_wave/file_card_uploader_spec.rb

```diff
@@ -0,0 +1,67 @@
+# -*- encoding : utf-8 -*-
+
+describe CarrierWave::FileCardUploader do
+  def test_file
+    File.new(File.join(FIXTURES_PATH, "file1.txt"))
+  end
+
+  def create_file_card storage_type, file=test_file, opts={}
+    Card::Auth.as_bot do
+      Card.create! opts.reverse_merge(name: "file card",
+                                      type_id: Card::FileID,
+                                      file: file,
+                                      storage_type: storage_type,
+                                      codename: "file_card_codename")
+    end
+  end
+
+  let(:local_file) { create_file_card :local }
+  let(:coded_file) { Card[:logo] }
+  let(:web_file) { create_file_card :web, "http://web.de/test.txt" }
+
+  describe "#db_content" do
+    context "coded file" do
+      subject { coded_file }
+      it "returns correct identifier" do
+        expect(subject.attachment.db_content)
```

Comment:
something about this api seems funny to me:

`subject.attachment.db_content`

my instinct is that the card, not the uploader, should have db_content.  but I get what you're going for.  No change to recommend yet.